README.md

```diff
@@ -1,4 +1,4 @@
-# SqlFormatter
+# Doctrine SqlFormatter
```

Comment:
Unrelated change?

Reply:
I won't review a "various improvements" PR and I think you know that.

Reply:
So tell me how many PRs you want from the well self contained commits.

Reply:
I don't have time for this, sorry.

Reply:
With all the honesty to you, there are 6 commits and I belive we do not deserve 6 PRs. So in order to get this PR passing for both of us, I need your input here.

Reply:
And you got your input.